dapr/clients/grpc/client.py

```diff
@@ -809,3 +810,22 @@ def wait(self, timeout_s: float):
                     if remaining < 0:
                         raise e
                     time.sleep(min(1, remaining))
+
+    def shutdown(
+            self,
+            metadata: Optional[MetadataTuple] = None) -> DaprResponse:
+        """Shutdown the sidecar.
+
+        This will ask the sidecar to gracefully shutdown.
+
+        The example shutdown the sidecar:
+
+            from dapr.clients import DaprClient
+
+            with DaprClient() as d:
+                resp = d.shutdown()
+        """
```

Comment:
```suggestion

        Args:
            metadata (tuple, optional): custom grpc metadata
        """
```

Though I'm not sure it is strictly necessary to take metadata as a parameter at all here. We don't do anything with it on the runtime side, correct?

Reply:
I was not sure for the metadata, does the auth token (https://docs.dapr.io/operations/security/api-token/) can go through those metadata ? Or it's managed by the `DaprClient` ?

Reply:
We have an interceptor add the token to every request: 
https://github.com/dapr/python-sdk/blob/master/dapr/clients/grpc/client.py#L89-L93

The user shouldn't need to pass anything themselves.